<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UploadRDFDialog</class>
 <widget class="QDialog" name="UploadRDFDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="tripleStoreURL">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Triple Store URL:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tripleStoreURLEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="tripleStoreGraph">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Triple Store Graph:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tripleStoreGraphEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="username">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>95</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="password">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>95</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="checkConnectionButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Check Connection</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
